dowhy/causal_estimator.py

```diff
@@ -214,9 +244,9 @@ def get_confidence_intervals(self, estimate):
         # Now use the data obtained from the simulations to get the value of the confidence estimates
         # Sort the simulations
         simulation_results.sort()
-        # Now we take the 5th and the 95th values
-        lower_bound_index = int( 0.05 * len(simulation_results) ) 
-        upper_bound_index = int( 0.95 * len(simulation_results) )
+        # Now we take the pth and the (1-p)th values, where p is the chosen confidence level
```

Comment:
need to change based on the revised meaning of confidence level, typically reported as 0.95 not 0.05.

Reply:
Yes, I'll make that change.